### DISCUSSION TITLE
"File is too small" problem.

### DISCUSSION THREAD

**Discussion by enzomarin97** [community/community#36846] [GitHub Education]
### For days I have this problem when uploading the proof of academic status. My file is JPEG , 1240x1755, 276.6 KB. I would appreciate any solution.
![brave_MmbXfXYVse](https://user-images.githubusercontent.com/32345153/197081847-8b496c2b-459e-4752-b90a-cc140c5e6618.png)

**Comment by dbayterv** [comment]
Hi Enzo. You can resize your image... https://www.iloveimg.com/es/redimensionar-imagen/jpg-cambiar-tamano

**Comment by kude5n1k** [comment]
Just try to increase the size of the image through any online service

**Comment by ossanna16** [comment]
Thank you for asking a good question @enzomarin97, I'm glad you found our community 🙂
